USE-CASES/bullet chatting rendering.md

```diff
@@ -27,28 +27,29 @@ When watching a video, the viewers may see the bullet chatting scrolling or bein
 
 ### Expected Data:
 
-Data structure to render a bullet chatting, such as color, fontSize, animating duration, image. 
+Data structure to render a bullet chatting, such as color, font size, animation duration, images etc.
 
 ### Dependencies - Affected Bullet Chatting deliverables and/or work items:
 
 ### Description:
 
 There are some details covered in this use case 
 
-- The bullet chatting content includes text, images and animation, etc. 
+- The bullet chatting content can include both text and images.
+- The bullet chatting can be animated.
 - The bullet chatting can be statically displayed at the top or bottom of the video area.
 - The bullet chatting can be displayed in the video area by scrolling in a fixed direction, such as scrolling from right to left.
-- If the user pause the video, and the bullet chatting is also paused.
+- If the user pauses the video, and the bullet chatting is also paused.
 - The bulelt chatting is in sync with the video timeline. Just like WebVTT, when the video timeline changes, the bullet chatting will change simultaneously as well.
-- When the user hover on a specified bullet chatting, the bullet chatting may stop scrolling, and other bullet chatting is display as usual. This behavior may be not a default way, but the ability to control this behaviour should be provided.
-- The bullet chatting can be overlaped.
+- When the user hover on a specified bullet chatting, the bullet chatting may stop scrolling, and other bullet chats are rendered as usual. This behavior may not be the default, but the ability to control this behavior should be provided.
+- The bullet chatting can be overlapping behavior can be controlled.
 
 #### Variants:
 
 
 ### Gaps:
 
-When syncing with the video timeline, service provider need to know current video time, but [timeupdate](https://html.spec.whatwg.org/multipage/media.html#event-media-timeupdate) DOM event is triggered every 250 millisecond, a more accurate time need to be provided. [requestAnimationFrame()](requestAnimationFrame()) may be good option, but it can be more better when considering to use [video.requestVideoFrameCallback()](https://github.com/WICG/video-rvfc/blob/gh-pages/explainer.md).
+When syncing with the video timeline, service provider need to know current video time, but since [timeupdate](https://html.spec.whatwg.org/multipage/media.html#event-media-timeupdate) DOM event is triggered every 250 millisecond, a more accurate time need to be provided. [requestAnimationFrame()](https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#animation-frames) may be good option, but it can be more better when considering to use [video.requestVideoFrameCallback()](https://github.com/WICG/video-rvfc/blob/gh-pages/explainer.md).
```

Comment:
Suggested change: When syncing with the video timeline, the service provider needs to know the current video time, but since timeupdate DOM events are triggered every 250 milliseconds, a more accurate time needs to be provided.